account_invoice_overdue_reminder/wizard/overdue_reminder_wizard.py

```diff
@@ -0,0 +1,599 @@
+            'interface': self.interface,
+            }
+        return vals
+
+
+class OverdueReminderStartPayment(models.TransientModel):
+    _name = 'overdue.reminder.start.payment'
+    _description = 'Status of payments'
+
+    wizard_id = fields.Many2one(
+        'overdue.reminder.start', ondelete='cascade')
+    journal_id = fields.Many2one(
+        'account.journal', string='Journal', readonly=True)
+    last_entry_date = fields.Date(
+        string='Last Entry', readonly=True)
+    last_entry_create_date = fields.Datetime(
+        string='Last Entry Created on', readonly=True)
+    last_entry_create_uid = fields.Many2one(
+        'res.users', string='Last Entry Created by', readonly=True)
+
+
+class OverdueReminderStep(models.TransientModel):
+    _name = 'overdue.reminder.step'
+    _description = 'Overdue reminder wizard step'
+
+    partner_id = fields.Many2one(
+        'res.partner', required=True, string='Invoicing Contact')
+    partner_email = fields.Char(related='partner_id.email', readonly=True)
+    partner_phone = fields.Char(related='partner_id.phone', readonly=True)
+    partner_mobile = fields.Char(related='partner_id.mobile', readonly=True)
+    commercial_partner_id = fields.Many2one(
+        'res.partner', string='Customer', readonly=True, required=True)
+    user_id = fields.Many2one('res.users', required=True, readonly=True)
+    counter = fields.Integer(string="New Remind Counter", readonly=True)
+    date = fields.Date(default=fields.Date.context_today, readonly=True)
+    reminder_type = fields.Selection(
+        '_reminder_type_selection', default='mail',
+        string='Reminder Type', required=True)
+    mail_cc_partner_ids = fields.Many2many('res.partner', string='Cc')
+    mail_subject = fields.Char(string='Subject')
+    mail_body = fields.Html()
+    result_id = fields.Many2one(
+        'overdue.reminder.result', string='Call Result/Info')
+    result_notes = fields.Text(string='Call Notes')
+    create_activity = fields.Boolean()
+    activity_type_id = fields.Many2one(
+        'mail.activity.type', string='Activity')
+    activity_summary = fields.Char(string='Summary')
+    activity_deadline = fields.Date('Deadline')
+    activity_note = fields.Html(string='Note')
+    activity_user_id = fields.Many2one(
+        'res.users', string='Assigned to', default=lambda self: self.env.user)
+    letter_printed = fields.Boolean(readonly=True)
+    invoice_ids = fields.Many2many(
+        'account.invoice', string='Overdue Invoices', readonly=True)
+    company_id = fields.Many2one(
+        'res.company', readonly=True, required=True,
+        default=lambda self: self.env['res.company']._company_default_get())
+    warn_unreconciled_move_line_ids = fields.Many2many(
+        'account.move.line', string='Unreconciled Payments/Refunds',
+        readonly=True)
+    unreconciled_move_line_normal = fields.Boolean(
+        string='Check if unreconciled payments/refunds above have a good '
```

Comment:
```suggestion
        string='To check if unreconciled payments/refunds above have a good '
```
String change to match the UI context